clustering/infinispan/extension/src/test/java/org/jboss/as/clustering/infinispan/subsystem/TransformersTestCase.java

```diff
@@ -263,8 +290,10 @@ private static FailedOperationTransformationConfig createFailedOperationConfig(M
         if (InfinispanModel.VERSION_2_0_0.requiresTransformation(version)) {
             for (PathElement path : Arrays.asList(DistributedCacheResourceDefinition.WILDCARD_PATH, ReplicatedCacheResourceDefinition.WILDCARD_PATH)) {
                 PathAddress cacheAddress = containerAddress.append(path);
-                config.addFailedAttribute(cacheAddress.append(BackupResourceDefinition.WILDCARD_PATH), FailedOperationTransformationConfig.REJECTED_RESOURCE);
-                config.addFailedAttribute(cacheAddress.append(BackupForResourceDefinition.PATH), FailedOperationTransformationConfig.REJECTED_RESOURCE);
+                config.addFailedAttribute(
+                        cacheAddress.append(BackupsResourceDefinition.PATH).append(BackupResourceDefinition.WILDCARD_PATH), FailedOperationTransformationConfig.REJECTED_RESOURCE);
+                config.addFailedAttribute(
+                        cacheAddress.append(BackupForResourceDefinition.PATH), FailedOperationTransformationConfig.REJECTED_RESOURCE);
```

Comment:
The reject xml should not use statistics-enabled=true, and this should be added as a failed attribute in the relevant cases.